pulsar-client-cpp/lib/HTTPLookupService.cc

```diff
@@ -195,6 +204,7 @@ namespace pulsar {
         LookupDataResultPtr lookupDataResultPtr = boost::make_shared<LookupDataResult>();
         lookupDataResultPtr->setBrokerUrl(brokerUrl);
         lookupDataResultPtr->setBrokerUrlSsl(brokerUrlSsl);
+        LOG_INFO("parseLookupData = "<<*lookupDataResultPtr);
```

Comment:
Agreed but lookups are _not_ so frequent in normal cases and even during disconnection they quickly get ratelimited to once per minute.

By adding more debugs I just want to make sure we have a better understanding of what is happening during the incident - since Curl HTTP Lookup is the one of the new things we have added in this release.